spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java

```diff
@@ -348,6 +383,59 @@ private static Route route(
                     .build();
     }
 
+    private static void addAdditionalPath(ServerBuilder sb, List<Integer> exposedPorts,
+                                          ExposableWebEndpoint endpoint,
+                                          SimpleHttpCodeStatusMapper statusMapper,
+                                          @Nullable CorsServiceBuilder cors, HealthEndpointGroups groups) {
+        for (WebOperation operation : endpoint.getOperations()) {
+            final WebOperationRequestPredicate predicate = operation.getRequestPredicate();
+            final String matchAllRemainingPathSegmentsVariable =
+                    predicate.getMatchAllRemainingPathSegmentsVariable();
+            // group operation has matchAllRemainingPathSegmentsVariable.
+            // e.g. /actuator/health/{*path}
+            // We can send a request to /actuator/health/foo if the group name is foo.
+            //
+            // We have to check if the group has additional path or not.
+            // e.g. management:
+            //        endpoint:
+            //          health:
+            //            group:
+            //              foo:
+            //                include: ping
+            //                additional-path: "management:/foohealth"
+            if (matchAllRemainingPathSegmentsVariable != null) {
+                if (!exposedPorts.isEmpty()) {
+                    final Set<HealthEndpointGroup> additionalGroups = groups.getAllWithAdditionalPath(
+                            WebServerNamespace.MANAGEMENT);
```

Comment:
I don't know what is the minimum supported version of Spring Boot 2.x.
The following classes newly imported are introduced after 2.0.0
- 2.6.0 - `AdditionalHealthEndpointPath`, `WebServerNamespace`, `OperationArgumentResolver`
- 2.4.0 - `ApiVersion`
- 2.2.0 - `HealthEndpointGroup`, `HealthEndpointGroups`
I guess we can see `ClassNotFoundException` if some users use Armeria 1.14.0 with Spring Boot 2.5.x or lower.

Reply:
Had a chat with @ikhoon and we have decided to support the backward compatibility from 2 versions older(in minor) than the current version. e.g. if we use Spring Boot 2.6.0, we guarantee compatibility from 2.4.0.
It's because Spring usually removes the classes, which are deprecated after 2 versions upgrade and I think two versions are enough to give users room to upgrade the Spring Boot version.